app/build.gradle

```diff
@@ -11,6 +11,7 @@ def versions = rootProject.ext.versions
 repositories {
     maven { url "https://oss.sonatype.org/content/repositories/snapshots" }
     mavenCentral()
+    jcenter()
```

Comment:
do you need to call it from this file? don't you inherit from root build.gradle?

Reply:
I did not check it. But I'm going to have a look :)